components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -2886,35 +2890,15 @@ def listColleagues(self):
         """
 
         default = self.getObject(
-            "ExperimenterGroup", self.getEventContext().groupId)
+            "ExperimenterGroup", self.getEventContext().groupId,
+            opts={'load_experimenters': True})
         if not default.isPrivate() or self.isLeader():
             for d in default.copyGroupExperimenterMap():
                 if d is None:
                     continue
                 if d.child.id.val != self.getUserId():
                     yield ExperimenterWrapper(self, d.child)
 
-    def groupSummary(self, gid=None, exclude_self=False):
```

Comment:
Apologies, you're of course right. I had confused this with https://github.com/openmicroscopy/openmicroscopy/pull/5402/files#diff-5586c69038567a9c1960ae4f6d731278L509 where I convinced myself that the removal was less critical since it was in `webclient`